src/web/mjs/connectors/AllManga.mjs

```diff
@@ -1,12 +0,0 @@
-import WordPressMadara from './templates/WordPressMadara.mjs';
-
-export default class AllManga extends WordPressMadara {
-
-    constructor() {
-        super();
-        super.id = 'allmanga';
-        super.label = 'مانجا العرب (M ARAB)';
-        this.tags = [ 'manga', 'arabic' ];
-        this.url = 'https://mangaarab.com';
```

Comment:
Could it be moved to https://www.manhwa.ae, or is this a different website...

Reply:
I never saw this site, I can only say that https://www.manhwa.ae is a WordpressMadara site. From the website title and URL alone I would say it is a different site. To be safe I propose to add  https://www.manhwa.ae while still remove 'allmanga'?

Reply:
I vote for just removal, user may request Manhwa.ae when needed